#12 Remove useless variable

kreghek opened the pull request:
#12

kreghek committed:
#12 Remove useless variable

CLAassistant commented:
[![CLA assistant check](https://cla-assistant.io/pull/badge/not_signed)](https://cla-assistant.io/tektronix/initialstate.NET?pullRequest=13) <br/>Thank you for your submission, we really appreciate it. Like many open source projects, we ask that you sign our [Contributor License Agreement](https://cla-assistant.io/tektronix/initialstate.NET?pullRequest=13) before we can accept your contribution.<br/><sub>You have signed the CLA already but the status is still pending? Let us [recheck](https://cla-assistant.io/check/tektronix/initialstate.NET?pullRequest=13) it.</sub>

kreghek committed:
Remove useless initialization

https://app.codacy.com/manual/Tektronix/initialstate.NET/issues/index

Remove this initialization, the compiler will do that for you.

dwyban reviewed (changes requested):
This PR #13 along with PR #14 #15 #16 are all simple code cleanup changes.  Please combine these into a single pull request.
